docs/spec_draft.md

```diff
@@ -670,6 +718,57 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.remainder
+
+`stablehlo.remainder(lhs, rhs) -> result`
+
+### Semantics
+
+Performs element-wise remainder of dividend `lhs` and divisor `rhs` tensors and
+produces a `result` tensor. The sign of the result is taken from the dividend,
+         and the absolute value of the result is always less than the divisor's
+         absolute value. The remainder of the any division operation `x/y` is
+         calculated as `x - n*y`, where `n` is `x/y` with its fractional part
+         truncated. For floating-point element types, this is in contrast with
```

Comment:
Consider rephrasing so that this uses the names introduced above, e.g. "The result calculated as \`lhs - d*rhs\`, where \`d = stablehlo.divide(lhs, rhs)\`".